normalization/ufonormalizer.py

```diff
@@ -1334,9 +1335,8 @@ def subpathReadFile(ufoPath, *subpath):
     Read the contents of a file.
     """
     path = subpathJoin(ufoPath, *subpath)
-    f = open(path, "rb")
-    text = f.read()
-    f.close()
+    with open(path, "r") as f:
```

Comment:
you should specify the encoding="utf-8" here, otherwise io.open will default to the preferred locale encoding, which on windows is not utf-8.